### PR TITLE
add method to update and save rubric on ev closing

### PR DESCRIPTION
**Pull request by CharlotteSpinks**

### CODE REVIEW

File: lib.php [hunk 1]
```diff
@@ -1932,6 +1932,49 @@ public function sync_tii_assignment($cm, $coursetiiid, $workflowcontext = "site"
         }
     }
 
+    public function update_rubric_from_tii($cm) {
```
Comment (dwinn) [inline lib.php:1935]: Can we add a PHPdoc? Self explanatory what the method does but Moodle guidelines say methods should be documented.

**Review by dwinn — APPROVED** [review]
Looks good!

File: lib.php (hunk 1 above)
Comment (CharlotteSpinks) [reply lib.php:1938]: Done that :)

**Review by jmcgettrick — APPROVED** [review]

**Comment by jmcgettrick** [comment]
Verified.

**Pull request merged**